packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -1,4 +1,78 @@
-{{! ADD YOUR TEMPLATE CONTENT HERE }}
 <div class={{this.classNames}} ...attributes>
-  {{yield}}
-</div>
\ No newline at end of file
+  {{!-- {{yield}} --}}
+
+  {{! Hds-TabList }}
+  <div class="hds-tab-list">
```

Comment:
This is going to be separated out as a child component so I didn't want to use BEM naming style. I would have called it "hds-tabs-list" with an "s" but I was trying to follow naming in @heatherlarsen's CRD: https://docs.google.com/document/d/10OIZG32JaKxAf4uyuDjTU7SGGnwHqabBhPvfHQ8_ruA/edit#heading=h.gjdgxs

Reply:
@KristinLBradley to clarify this is not "my" CRD. The intention is that the CRDs are a collaboration between Designer and Engineer. One individual starts it (doesn't matter who), the other makes changes or reviews it and comments (whatever your preferred style is), and we discuss. 🙂 

I used `tablist` in the RFC because it's the semantic name for the role.

Reply:
Thanks, the naming in the CRD makes sense so I tried to follow it.